pyvista/core/filters/data_set.py

```diff
@@ -2445,12 +2537,13 @@ def streamlines(dataset, vectors=None, source_center=None,
             source.SetPoint1(pointa)
             source.SetPoint2(pointb)
             source.SetResolution(n_points)
+            _update_alg(source, progress_bar, 'Making Line')
```

Comment:
We don't need a progress bar here.  It's just a single line and will compute instantly.

Reply:
```suggestion
```